packages/twenty-server/src/engine/core-modules/environment/converters/config-var-value.converter.ts

```diff
@@ -0,0 +1,83 @@
+import { EnvironmentVariables } from 'src/engine/core-modules/environment/environment-variables';
+import { TypedReflect } from 'src/utils/typed-reflect';
+
+/**
+ * Handles type conversions for environment variables
+ * when storing and retrieving them from the database.
+ */
+export class ConfigVarValueConverter {
```

Comment:
Everywhere in this PR, don't hesitate to break your code into small functions, avoid else/elseif structure and do small subfunctions with early returns, that makes the code easy to read and remove the need for comments